docs/self-hosting.md

```diff
@@ -2,6 +2,16 @@
 
 Welcome to the self-hosting guide for Discuit! Here, you'll find step-by-step instructions on how to set up and run Discuit on your own server.
 
+## Prerequisites
+
+- [Git](https://git-scm.com/)
+
+## Cloning the Repository
+
+```shell
+git clone https://github.com/discuitnet/discuit.git && cd discuit
+```
+
```

Comment:
You can probably merge this into one section. I don't think we need to explicitly mention Git, but if you are, it should be mentioned in a single statement like "You can use [Git](https://git-scm.com) to clone the repository."

Reply:
```
## Cloning the Repository

You can use [Git](https://git-scm.com/) to clone the repository. Run the following command in your terminal:

\```shell
sudo apt install git
git clone https://github.com/discuitnet/discuit.git && cd discuit
\```
```

Would this look good? Or maybe make the header it self a link?

Reply:
I don't think we need to include the Git installation step, because it can vary between platforms, and is usually preinstalled. Plus, Debian/Ubuntu repos are very out of date anyway, and the Git website probably has the up-to-date installation instructions.

The second sentence is also not needed, you can just make it something like:

````md
## Cloning the repository

You can use [Git](https://git-scm.com/) to clone the repository:

```shell
git clone https://github.com/discuitnet/discuit.git && cd discuit
```
````